### PR TITLE
fix mongodb_uri in .env.local_dev

### PR DESCRIPTION
**Pull request by bandundu**
Changed the .env.local_dev file to prevent the OpenSignServer-container to be unable to reach mongo db

### CODE REVIEW

**Comment by andrew-opensignlabs** [comment]
@rishabjasrotia can you please review this?

File: .env.local_dev [hunk 1]
```diff
@@ -23,6 +23,7 @@ appName=open_sign_server
 MASTER_KEY=XnAadwKxxByMr
 # Mongodb URI to connect to
 MONGODB_URI=mongodb://host.docker.internal:27017/OpenSignDB
```
Comment (bandundu) [inline .env.local_dev:25]: this line should be removed obviously

**Comment by rishabjasrotia** [comment]
Hello @bandundu 
I don't understood why host.docker.internal is not working.
May be you are using linux machine.
So in case of linux machine we should have add documentation  for your changes.

We do not need to alter .env values you can update installation docs for linux users.

**Comment by bandundu** [comment]
Hi @rishabjasrotia 
could very well be that this is then not a code issue but rather documentation.

Thank you for the feedback then!

**Pull request closed without merging**